packages/cli/src/commands/stream/update.ts

```diff
@@ -0,0 +1,85 @@
+import { flags } from '@oclif/command'
+import { TileDocument } from '@ceramicnetwork/stream-tile'
+import type { StreamMetadata } from '@ceramicnetwork/common'
+
+import { Command } from '../../command'
+import type { CommandFlags } from '../../command'
+import { parseControllers } from '../../utils'
+
+type Flags = CommandFlags & {
+  controller?: string
+  metadata?: StreamMetadata
+}
+
+export default class Update extends Command<
+  Flags,
+  {
+    content: string
+    streamId: string
+  }
+> {
+  static description = 'Update a stream'
+
+  static args = [
+    {
+      name: 'streamId',
+      description: 'Document StreamID',
+      required: true,
+    },
+    {
+      name: 'content',
+      description: 'Document Content',
+      required: true,
+      parse: JSON.parse,
+    },
+  ]
+  static flags = {
+    ...Command.flags,
+    controller: flags.string({
```

Comment:
It's confusing to have both `controller` and `metadata` flags I think, let's only have `metadata` as it already supports changing the controller